src/view-backend.c

```diff
@@ -146,6 +146,13 @@ wpe_view_backend_get_activity_state(struct wpe_view_backend* backend)
     return backend->activity_state;
 }
 
+void
+wpe_view_backend_dispatch_set_scale(struct wpe_view_backend* backend, float scale)
+{
+    if (backend->backend_client)
```

Comment:
I think we should check here that backend->backend_client->set_device_scale_factor is not null.

Reply:
Can do. I've re-set the branch to 1.2 as master had a merge conflict with `CMakeLists.txt` and 1.2 seemed to be more up-to-date, happy to push an updated diff against master with the `CMakeLists.txt` changes reverted. Obviously as you point out we need to get the WebKit patch committed too.